packages/content-type-remote-attachment/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 XMTP Labs (xmtp.com)
```

Comment:
I'd like to suggest that we change this to reflect the XMTP developer community instead of the XMTP Labs, the company. We should make this change across all of the open source repos.

```suggestion
Copyright (c) 2023 XMTP developers (xmtp.org)
```